### PR TITLE
Day 4 Task

### PR DESCRIPTION
**Pull request by PrathameshMahadik**
Completed Day 4 Tasks

### CODE REVIEW

File: feature/day4/Task3.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// 3. Write a program to give count of variables in a string ("abcabcdabbcc" => "a3b4c4d1")
+const input = "abcabcdabbcc";
+let result = "";
+let arrInput = input.split("").sort();
+let count = 1;
+for (let i = 1; i <= arrInput.length; i++) {
+  if (arrInput[i] === arrInput[i - 1]) {
+    count++;
+  } else {
+    result += arrInput[i - 1];
```
Comment (Pratik-Parate) [inline feature/day4/Task3.js]: here you can use
result += arrInput[i - 1] + count;

File: feature/day4/Task5.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// 5. Write a program to convert given string in camel case ("hello john doe" => helloJohnDoe)
+const input = "hello john doe";
+let output = "";
+let arrInput = input.split(" ");
+output += arrInput[0].toString();
+for (let i = 1; i < arrInput.length; i++) {
+  let str = arrInput[i].toString();
+  let ch = str.charAt(0);
+  ch = ch.toUpperCase();
```
Comment (Pratik-Parate) [inline feature/day4/Task5.js]: here you can use  const ch = str.charAt(0).toUpperCase();

File: feature/day4/Task7.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+// 7. Write a program to find index of duplicate elements in an array
+let arr = [10, 20, 30, 10, 30, 40, 50, 60, 10, 60];
+let index = [];
+var count = 0;
```
Comment (Pratik-Parate) [inline feature/day4/Task7.js]: what is count doing in this program?
remove count and without using Set() solve the problem

File: feature/day4/Task9.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+// 9. Write a program to find the last duplicate index in an array
+let arr = [10, 20, 30, 10, 30, 40, 50, 60, 10, 60];
+let index = [];
+var count = 0;
```
Comment (Pratik-Parate) [inline feature/day4/Task9.js]: what is count doing in this program?
remove count and without using Set() solve the problem

File: feature/day4/Task3.js (hunk 1 above)
Comment (PrathameshMahadik) [reply feature/day4/Task3.js]: Done

File: feature/day4/Task5.js (hunk 2 above)
Comment (PrathameshMahadik) [reply feature/day4/Task5.js]: Done

**Review by Pratik-Parate — APPROVED** [review]

**Pull request merged**